src/hyperion/external_interaction/ispyb/store_datacollection_in_ispyb.py

```diff
@@ -34,7 +36,6 @@
 
 I03_EIGER_DETECTOR = 78
 EIGER_FILE_SUFFIX = "h5"
-VISIT_PATH_REGEX = r".+/([a-zA-Z]{2}\d{4,5}-\d{1,3})(/?$)"
 
 
 class IspybIds(BaseModel):
```

Comment:
I can't comment on the exact location because it's not in the diff but `StoreInIspyb.experiment_type` is defined twice in its `__init__`